src/test/java/edu/hm/hafner/analysis/parser/violations/JUnitAdapterTest2.java

```diff
@@ -0,0 +1,30 @@
+package edu.hm.hafner.analysis.parser.violations;
+
+import edu.hm.hafner.analysis.AbstractParserTest;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+import edu.hm.hafner.analysis.assertj.SoftAssertions;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class JUnitAdapterTest2 extends AbstractParserTest {
```

Comment:
How is this test different from the previous one? If it is really different, then please add it as a second test case to class `JunitAdapterTest` and describe the difference in the JavaDoc.:
```
    @Test
    void should***MeaningfulName****() {
        Report report = parse("filename.txt");

        assertThat(report).hasSize(1);
    }

```

Reply:
I have added the second test case, according to what you instructed me.